src/Http/ServerRequest.php

```diff
@@ -1396,7 +1396,7 @@ protected function _parseAcceptWithQualifier($header)
      *
      * @param string|null $name Query string variable name or null to read all.
      * @return string|array|null The value being read
-     * @deprecated 3.4.0 Use getQuery() and withQueryParams() instead.
+     * @deprecated 3.4.0 Use getQueryParams() and withQueryParams() instead.
```

Comment:
I've updated this way because `getQueryParams()` is a direct replacement for the deprecated method, whereas `getQuery()` is not.

I can update though, if it's a change you feel would be helpful. I'd rather keep the upgrade process for users as clear as possible, which will also help with automated updates such as search and replace.